gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -6662,13 +8507,20 @@
                         strength of mechanism.
 
                         Examples of attacks are buffer overflow attacks.</ns0:description>
-                    <ns0:check system="http://open-scap.org/page/SCE">
-                        <ns0:check-import import-name="stdout" />
-                        <ns0:check-content-ref href="./AslrCheck.sh" />
+                    <ns0:ident system="http://cyber.mil/legacy">V-203754</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/legacy">SV-203754</ns0:ident>
+                    <ns0:ident system="http://cyber.mil/cci">CCI-000366</ns0:ident>
+                    <ns0:fixtext fixref="F-203754r1_fix">Configure the operating system to meet the requirement.</ns0:fixtext>
+                    <ns0:fix id="F-203754r1_fix" />
+                    <ns0:check system="http://oval.mitre.org/XMLSchema/oval-definitions-5">
+                        <ns0:check-content-ref href="stub-oval.xml" />
```

Comment:
This drops the check that address space layout randomization is enabled entirely, and converts it to the stub-oval check, and causes `notchecked` results in oscap, and Not Reviewed results in STIG Viewer when imported there.

(One wrinkle is that the script that does the check needs to happen on a running system, because it's checking a sysctl value exported from the running kernel's `/proc/sys` interface. So we should encourage checking against a running container over a given image, I suppose.)